src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-baseAnalysis.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    ID of a CodeSonar analysis to use as a base for "Warning Count Increase" conditions.
+    <br/>
+    If this field is not specified, the plugin will use the analysis results corresponding to the latest successful Jenkins build (if present).
+</div>
```

Comment:
We will need to add a description of this new parameter to the documentation before merging.

Reply:
@AmyGale please note the changes to the README.

Reply:
I recommend calling this something else. "Base analysis" is established terminology in the context of incremental analysis.

Reply:
Changed to `comparisonAnalysis`